cluster_experiments/power_config.py

```diff
@@ -185,6 +189,9 @@ def __post_init__(self):
             if self._are_different(self.covariates, None):
                 self._set_and_log("covariates", None, "analysis")
 
+        if "segmented" in self.perturbator:
```

Comment:
do you think it makes sense to raise this in the perturbator init? then we would raise when this is initialized from config or using the regular init

Reply:
I'm not sure: The difference is that in the specific perturbator class, the argument in question is already _not optional_, but it _is optional_ in the power_config if you choose not to use this specific perturbator.
So it feels like the error belongs more to the power_config, since in the estimator you can just rely on the signature. But we can also make our own convention here if you prefer, I don't have a suuuper strong opinion:)

Reply:
you're right, fine in here